### ISSUE TITLE
DocumentCard: DocumentCardLocations don't have visual indicators specifying that they are links

### ISSUE THREAD

**Issue by khmakoto** [OfficeDev/office-ui-fabric-react#10659]
### Environment Information

- **Package version(s)**: 7.39.2

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/khmakoto/pen/BaBeywZ?&editable=true

#### Actual behavior:

The `DocumentCardLocation` component is a link but does not have and visual indication of being one until hovered, which makes it seem like regular text.

An example of this is the `Marketing Documents` link on the image below:

![image](https://user-images.githubusercontent.com/7798177/65803902-e949fd00-e134-11e9-852d-abd133107e89.png)

#### Expected behavior:

The `DocumentCardLocation` component looks like an actual actionable link. However, this styling is baked into the component, so I wonder if this is a decision by design and, if so, what's the reasoning behind it.

#### Notes:

Bug 1 in #10646 - DocumentCard A11y Pass

**Comment by micahgodbolt** [comment]
asked design for guidance

**Comment by betrue-final-final** [comment]
Sent a message to Dave Woods.

**Comment by woods-dave** [comment]
Location links on a card like this, should be semibold (weight 600) and themePrimary. This is the new fluent spec across SP cards

**Comment by khmakoto** [comment]
@woods-dave let me make those changes to the links, I'll send a PR shortly.
